src/main/java/use_cases/GameCreationInteractor.java

```diff
@@ -0,0 +1,177 @@
+package use_cases;
```

Comment:
This class is quite long, maybe consider pulling some parts out into another class (e.g.a BuildPlayers class that handles the functionality of buildPlayers and humanPlayerBuilder methods since they are quite similar). This can help adhere to SRP.

Reply:
I do agree that this class is quite long. However I don't really know how to extract it. The build methods are all private and humanPlayerBuilder only serves as a helper method to buildPlayers (handles the data access logic). I don't want the build methods to be public as they don't serve any other purpose than to be used by GameCreationInteractor. Im open to feedback on this.